engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -131,5 +132,6 @@ public void swapReadWriteBuffers() {
         WRITE_ONLY_GBUFFER.setFbo(fbo);
         fboLookup.put(READ_ONLY_GBUFFER.getName(), READ_ONLY_GBUFFER.getFbo());
         fboLookup.put(WRITE_ONLY_GBUFFER.getName(), WRITE_ONLY_GBUFFER.getFbo());
+        notifySubscribers();
```

Comment:
Yes, true. But since DefaultFBOs internal FBO reference is up to date just before task list is executed, this didn't cause any harm. On the other hand, there can be some unexpected bugs that are not currently found because of this. So I think it's better to move `notifySubscribers` after `updateDefaultFBOs()`.